src/components/main/flight/ReceiptWithFlightScreen.js

```diff
@@ -1,232 +1,257 @@
-import React from 'react'
-
+import React from "react";
 import {
   View,
   Text,
   StyleSheet,
-  Dimensions,
   TouchableOpacity,
-} from 'react-native'
-
-import { Ionicons, FontAwesome } from '@expo/vector-icons';
-
-import Dash from 'react-native-dash';
-
-import COLORS from '../../../assets/Colors.js';
+  Keyboard,
+  Image,
+  SafeAreaView,
+} from "react-native";
+import { verticalScale, moderateScale } from "react-native-size-matters";
+import { List, ListItem, Left, Right } from "native-base";
+import COLORS from "../../../assets/Colors.js";
+import MenuBar from "../MenuBar.js";
+import Auth from "@aws-amplify/auth";
+import i18n from "i18n-js";
 
-import MenuBar from '../MenuBar.js';
-
-import Auth from '@aws-amplify/auth';
-
-import i18n from 'i18n-js'
+const leaf = require("../../../assets/images/img_receipt_leaf_nobg.png");
```

Comment:
please replace the file "../../../assets/images/img_receipt_leaf.png" (the one with the background) with this image(the one without a background) while keeping the name (without _nobg).

Furthermore, the imagie itself will need to be replaced later on. Hopefully flavio's guy can supply us with a transparent background png that also includes the shadow.

Reply:
Please rename the file **/img_receipt_leaf.png**